qa/tasks/cephfs/test_exports.py

```diff
@@ -113,10 +114,45 @@ def test_export_pin(self):
         else:
             self._wait_subtrees(status, 0, [('/1', 0), ('/1/4/5', 1), ('/a', 1), ('/a/b/aa/bb', 0)])
 
-        # Test getfattr
-        self.assertTrue(self.mount_a.getfattr("1", "ceph.dir.pin") == "0")
-        self.assertTrue(self.mount_a.getfattr("1/4", "ceph.dir.pin") == "-1")
-        self.assertTrue(self.mount_a.getfattr("1/4/5", "ceph.dir.pin") == "1")
+    def test_getfattr(self):
```

Comment:
`test_getfattr` -> `test_export_pin_getfattr`

Reply:
Ack. I have updated it